### ISSUE TITLE
Comments not showing up in application

### ISSUE THREAD

**Issue by waldean** [RittmanMead/ChitChat#5]
Sometimes when adding comments under "Home" on a dashboard (not selecting any sub component of the dashboard page) the newly added comments "disappear". They appear in the database table (annotation) as expected but are nowhere to be seen in the UI while in BI.
![capture1](https://user-images.githubusercontent.com/43331708/45608092-5b373300-ba59-11e8-9821-96c09c58e1d1.JPG)
![capture2](https://user-images.githubusercontent.com/43331708/45608093-61c5aa80-ba59-11e8-83b0-9fb35aec684f.JPG)

**Comment by jiuks** [comment]
Thanks for letting us know. Do the comments reappear if you refresh the page?

**Comment by waldean** [comment]
No, unfortunately it doesn’t seem to make any difference. The comment "disappears" from the screen at the same moment as you post it. Then even if you refresh the screen it's still not visible (except as mentioned earlier in the db table iteself).

Kind regards
Anders
